csrc/host_ir/host_ir.cpp

```diff
@@ -32,13 +32,26 @@ NVFUSER_DEFINE_CLONE_AND_CREATE(HostUnit)
 
 std::string HostUnit::toString(int indent_size) const {
   std::stringstream ss;
+  ss << toInlineString(indent_size) << ": Inputs={";
+  std::for_each(
+      fusion_to_execute()->inputs().begin(),
+      fusion_to_execute()->inputs().end(),
+      [&ss](auto input) { ss << input->toString(0) << ", "; });
+  ss << "} -> Outputs={";
+  std::for_each(
+      fusion_to_execute()->outputs().begin(),
+      fusion_to_execute()->outputs().end(),
+      [&ss](auto output) { ss << output->toString(0) << ", "; });
+  ss << "}";
   fusion_->print(ss, false);
   return ss.str();
 }
 
 // TODO: implement better ?
 std::string HostUnit::toInlineString(int indent_size) const {
-  return toString(indent_size);
+  std::stringstream ss;
+  ss << "HostUnit" << name();
```

Comment:
```suggestion
  ss << "HostUnit " << name();
```

Nit: If/when names contain letters, the extra space will be important for readability. If you think names will always be integers, consider replacing that with `id()` that returns an integer.

Reply:
Can you provide a pointer to the method `id` you are talking about? I only see it in the context of "serde", I fail to understand how you suggest using that.

Anyways, I do think names will always be integers, that is why I think it's ok to not include space, similarly to what is done for TensorView
https://github.com/NVIDIA/Fuser/blob/d75fc9322d4bf9c9b69af5406e1e55cd1ee02af9/csrc/ir/utils.cpp#L812

But I have no strong opinion so please lmk what you think is best

Reply:
Actually `Statement::name()` always returns an int:
https://github.com/NVIDIA/Fuser/blob/d75fc9322d4bf9c9b69af5406e1e55cd1ee02af9/csrc/ir/base_nodes.h#L52

Reply:
> Anyways, Statement::name() actually always returns an int:

Never mind. Ideally, I'd call it `Statement::id()` because it always return an integer. But it would be a low ROI renaming refactor at this moment.